cli/ballerina-cli/src/main/resources/cli-help/ballerina-graph.help

```diff
@@ -0,0 +1,45 @@
+NAME
+       ballerina-graph - Print the dependency graph
+
+SYNOPSIS
+       bal graph [OPTIONS] [<package>|<source-file>]
+
+
+DESCRIPTION
+       Resolve the dependencies of the current package and print the final
+       dependency graph into the console.
+
+       This produces the textual representation of the dependency graph
+       using the DOT graph description.
+
+
+OPTIONS
+       --dump-raw-graphs
+          Print all the dependency graphs created in each attempt to update.
```

Comment:
I would write this: "Print all intermediate graphs created in the dependency resolution process."